src/k2/persistence/plog_client/PlogClient.cpp

```diff
@@ -169,20 +169,50 @@ seastar::future<std::tuple<Status, uint32_t>> PlogClient::seal(String plogId, ui
     }
 
     return seastar::when_all_succeed(sealFutures.begin(), sealFutures.end())
-        .then([this](std::vector<std::tuple<Status, dto::PlogSealResponse> >&& results) {
+        .then([this](std::vector<std::tuple<Status, dto::PlogSealResponse> >&& results) { 
             Status return_status;
             uint32_t sealed_offset;
             for (auto& result: results){
                 auto& [status, response] = result;
                 return_status = std::move(status);
                 sealed_offset = response.sealedOffset;
-                if (!return_status.is2xxOK())
+                if (!return_status.is2xxOK()) 
                     break;
             }
             return seastar::make_ready_future<std::tuple<Status, uint32_t> >(std::tuple<Status, uint32_t>(std::move(return_status), std::move(sealed_offset)));
         });
 }
 
+seastar::future<std::tuple<Status, std::tuple<uint32_t, bool>>> PlogClient::getPlogStatus(String plogId){

```

Comment:
this nested tuple should be a response type

Reply:
fixed